libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -394,11 +394,12 @@ protected void onAuthFlowComplete(TokenEndpointResponse tr) {
     private class SwapJWTForAccessTokenTask extends BaseFinishAuthFlowTask<LoginOptions> {
 
         @Override
-        protected TokenEndpointResponse performRequest(LoginOptions options) {
+        protected TokenEndpointResponse performRequest(LoginOptions options) throws Exception {
```

Comment:
We should remove `throws Exception` here, since we're catching it anyway and triggering `onAuthFlowError`.